dace/transformation/passes/lift_trivial_if.py

```diff
@@ -26,14 +27,56 @@ def should_reapply(self, modified: ppl.Modifies) -> bool:
     def depends_on(self):
         return {}
 
+    def _make_unique_names(self, sdfg: dace.SDFG):
+        all_blocks = {
+            n
+            for n, _ in sdfg.all_nodes_recursive()
+            if isinstance(n, dace.SDFGState) or isinstance(n, ControlFlowRegion) or isinstance(n, ControlFlowBlock)
+        }
+        all_labels = set()
+
+        def _find_new_name(cfg: ControlFlowRegion) -> str:
```

Comment:
Why not use data.find_new_name(cfg_label, all_labels)?

Reply:
Now, having a pass to replace those integer comparisons with proper booleans seem good to me, still I think I should have some fallback code just in case.